File: benchmark/results/java-multilang/run-004/b-baseline/output/com/metaengine/demo/analytics/services/EventRepository.java
package com.metaengine.demo.analytics.services;

import java.util.List;

import com.metaengine.demo.analytics.aggregates.Event;

/** EventRepository persistence service. */
public class EventRepository {

    /** Persist an Event from a partial input. */
    public Event create(Event input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Event by id (may return null). */
    public Event findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List events up to limit. */
    public List<Event> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Event by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-invocation-script/run-002/b-baseline/output/src/domain/analytics/services/EventRepository.ts
import { Event } from "../aggregates/Event";

/** Service EventRepository. */
export class EventRepository {
  create(input: Partial<Event>): Event {
    void input;
    throw new Error("not implemented");
  }
  findById(id: string): Event | null {
    void id;
    throw new Error("not implemented");
  }
  list(limit: number): Event[] {
    void limit;
    throw new Error("not implemented");
  }
  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}
